README.md

```diff
@@ -93,3 +93,63 @@ Understands the following events:
 </pre></td>
   </tr>
 </table>
+
+## Linking local web-workspace-navigator into node_modules
+
+In order to work with non published artifacts of web-workspace-navigator within the test-editor-web, linking can be used.
+Given the following variables, this are the steps to have functional linking in place without modifying any sources.
+TODO: if deemed useful these steps should be put into scripts.
+
+``` shell
+WORKSPACE_NAVIGATOR="~/path/to/project"
+TEST_EDITOR_WEB="~/path/to/project"
+PREFIX=`npm prefix -g`
+```
+
+### bring linking into place
+
+``` shell
+cd $WORKSPACE_NAVIGATOR
+npm link
+cd $TEST_EDITOR_WEB
+npm link @testeditor/workspace-navigator
+cd $PREFIX/lib/node_modules/@testeditor
+rm workspace-navigator # old linking
+ln -s $WORKSPACE_NAVIGATOR/out-tsc/lib-es5/ workspace-navigator # new linking
```

Comment:
… then isn't this undoing all of that?
In other words, if we put our own symlink in place here, aren't the two calls to `npm link` completely redundant? Or does it have some other relevant side effect that we depend on?